### ISSUE TITLE
Setting the line and frame trigger pulse widths set to the RPi

### ISSUE THREAD

**Issue by eefer** [eefer/DE2115_MultiLockins#4]
As discussed in our conference call on 12/5/19, we think it would be easier on a user if the FPGA made the line and frame trigger pulse widths to be exactly as the RPi needs to register those triggers. We discussed making it always 1.5ms or something. The idea is that if the FPGA registered a pulse, then that pulse would be registered by the RPI regardless of the pulse length sent to the FGPA. This sounds like something Bill and/or Saurabh could address.

**Comment by wilsonjwcsu** [comment]
Just to refresh my memory and get on the same page..

- Are we talking about the AFM sync Schmidt trigger?
- Was this issue partially addressed by the features of SW[17] and SW[16]?

**Comment by eefer** [comment]
Yes I believe it is the AFM sync Schmidt trigger. This was addressed somewhat by the features of those switches, but I found when I connected the MAFLIA system to my AFM, even with those switches engaged it wasn't enough to make the trigger pulses long enough to be registered by the RPi. We had discussed just making the FPGA take whatever pulse it gets from the AFM that turning it into a pulse that the RPi can automatically read (right pulse width and height) instead of trying to use or modify the signal coming from the AFM, since that signal will vary quite a bit from AFM to AFM.
